### PR TITLE
Use dirs for placing the config in $XDG_HOME/.config

### PR DESCRIPTION
**Pull request by dsully**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -54,7 +55,12 @@ impl App {
 
 fn run(run_interactive: bool) -> bool {
     println!("{} Starting!", PREFIX);
-    let config: Config = toml::from_str(&fs::read_to_string("config.toml").unwrap()).unwrap();
+
+    let mut config_path = dirs::config_dir().unwrap();
+    config_path.push("topclean.toml");
```
Comment (mertd) [inline src/main.rs:60]: I think this line needs to push `topclean` if the resulting path is supposed to be `$XDG_HOME/.config/topclean/config.toml`.

**Review by mertd — COMMENTED** [review]
Thank you for the PR @dsully!

I interpret these changes as moving the default location for the `config.toml`. May I ask your motivation behind this?

Since this config.toml currently only contains a list of supported apps and their clean commands, I actually thought about going the opposite direction and moving the app list back into an .rs file to allow bundling everything together into one executable (https://github.com/mertd/topclean/issues/17).

I do agree that for a general config.toml file that contains actual configuration (e.g. the switch to skip interactive commands), we could use your idea.

**Comment by dsully** [comment]
If you're just going to make the config not configurable, then this doesn't make sense. :)

**Comment by mertd** [comment]
I think it makes sense like this:
1. rename config.toml to apps.toml, update main.rs accordingly
2. read options from cli arguments (1. priority) and optional config.toml (2. priority)

Currently, the only option is to skip commands that require user interaction to finish, but I am sure there will be more.

Let me know if you're interested in moving forward with the PR this way!

**Comment by dsully** [comment]
Sure

**Pull request closed without merging**